frontend/angular.json

```diff
@@ -76,10 +76,24 @@
               "src/styles.scss"
             ],
             "scripts": [
-              "node_modules/@panva/oauth4webapi/build/index.js"
+
```

Comment:
This library is required.

Reply:
I can add it back but it was throwing JS errors